compiler/rustc_target/src/spec/uefi_msvc_base.rs

```diff
@@ -43,7 +43,7 @@ pub fn opts() -> TargetOptions {
         exe_suffix: ".efi".to_string(),
         allows_weak_linkage: false,
         panic_strategy: PanicStrategy::Abort,
-        stack_probes: true,
+        stack_probes: StackProbeType::InlineOrCall { min_llvm_version_for_inline: (11, 0, 1) },
```

Comment:
I think this is where uefi should be changed to `Call` only.

Reply:
i.e. both "*-uefi" targets use a windows `llvm_target`, which returns false in LLVM's `hasInlineStackProbe`.

Reply:
Adjusted for this.